docs/_mar-milestone_.md

```diff
@@ -0,0 +1,43 @@
+# Monthly Milestone - March 2020
+
+## Overview
+
+- Migration guide: Stop starting, start finishing!
+
+## Goals
+
+- [ ] Migration guide https://github.com/strongloop/loopback-next/issues/3935
+
+  _General runtime_
+
+  - [ ] [5] Which official LB3 components are not planned in LB4 #3956 (from Feb
+        milestone)
+  - [ ] [8] Migration Guide: Discuss life cycle differences between LB3 and LB4
+        #3935 (from Feb milestone)
+  - [ ] [3] How to map LB3 CLI commands to LB4 CLI #3953 (from Feb milestone)
+  - [ ] [5] Spike: How to migrate LB3 components #4099
+
+  _Authentication & Authorization_
+
+  - [ ] [8][migration/auth] Create a guide to describe how to migrate LB3 User
+        based authentication to LB4 #4521
+  - [ ] [8] How to migrate apps using loopback-component-passport #3958
+  - [ ] [5] migration/auth] Describe how to simulate ACLs with LB4 authorization
+        #4522
+
+- [ ] Bugs
+  - [ ] Belongs-To relation tries to resolve for foreign key NULL #4372
+
+## Stretch Goals
+
+- [ ] Migration guide https://github.com/strongloop/loopback-next/issues/3935
+
+  - [ ] Spike on migrating Angular SDK client (1.x and 2+) (to be created -
+        originated from https://github.com/strongloop/loopback-next/issues/3954)
+
+- [ ] Spike: robust handling of ObjectID type for MongoDB #3456
+
+## Looking for community contributions
+
+_Community contributors: Want to contribute but don't know where to start? Here
+is our wish list for this month:_
```

Comment:
Model.toObject() should preserve prototypes (e.g. Date and ObjectID values) #3607

Reply:
Looking from the list of "good first issue" with highest number of upvotes: https://github.com/strongloop/loopback-next/issues?page=1&q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22+sort%3Areactions-%2B1-desc.

- Add support for "include" and "fields" to findById (REST API) https://github.com/strongloop/loopback-next/issues/1721
- Best practice for encrypting model properties https://github.com/strongloop/loopback-next/issues/2095 . _Not sure if this is only good for a very limited set of use cases._
- Complex OpenAPI Validations with `@property` https://github.com/strongloop/loopback-next/issues/1624

Reply:
The list proposed above looks good to me 👍